includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -884,6 +888,8 @@ public function get_settings_text( $settings_key, $default = false, $autop = tru
 		// fallback to first array element if default is not present
 		} elseif( ! empty( $setting ) && is_array( $setting ) ) {
 			$text = reset( $setting );
+		} else {
+			$text = $setting;
```

Comment:
May I ask why this part has been added here?

Reply:
@MohamadNateqi yes sure. The `get_settings_text` method gets the text in the specific language. In case we don't have a multilingual setup, the `$this->get_setting` method won't return an array, but one string value. Therefore, in case the `$setting` is not an array, assign it to the `$text` variable. In case the `$text` variable stays empty all the way, we will assign it to the default value provided through function parameter.

In short, I have modified this method so that it works in cases multilingual setups are not enabled.

Reply:
Thanks.